src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -745,12 +745,23 @@ size_t ShenandoahHeap::initial_capacity() const {
   return _initial_size;
 }
 
-bool ShenandoahHeap::is_in(const void* p) const {
+bool ShenandoahHeap::is_in_bounds(const void* p) const {
```

Comment:
Is this the same as is_in_reserved()? Do we even need that new method?

Reply:
Right, we can use `is_in_reserved` instead. In fact, our current method that computes this from the region sizes is not necessary, as the heap regions cover the heap exactly. So we can just as for `is_in_reserved` everywhere, without any loss.